package.json

```diff
@@ -73,7 +73,9 @@
     "react-instantsearch-dom": "^6.4.0",
     "react-popper": "^2.2.4",
     "react-select": "^3.1.0",
+    "remark": "^10.0.1",
```

Comment:
We're not exactly adding new dependencies here - these two packages are already being used by `gatsby-transformer-remark`